src/Libraries/RevitNodesUI/RevitDropDown.cs

```diff
@@ -562,7 +562,7 @@ public StructuralColumnTypes()
     }
 
     [NodeName("Spacing Rule Layout")]
-    [NodeCategory(BuiltinNodeCategories.GEOMETRY_CURVE_DIVIDE)]
+    [NodeCategory("Revit.Elements.DividedPath.Actions")]
```

Comment:
This is not what I meant, @aosyatnik. I would like you to make a new `NodeCategories.cs` in all three Revit branches. Then you can introduce `BuiltinNodeCategories.GEOMETRY_CURVE_DIVIDE` in that new file.